src/runtime/vm/vm.cc

```diff
@@ -537,7 +631,7 @@ void VirtualMachine::RunLoop() {
         from_obj = ReadRegister(instr.from);
         WriteRegister(instr.dst, from_obj);
         pc_++;
-        goto main_loop;
+        break;
```

Comment:
nit: This weird control flow may be the result of someone trying to improve I$ behavior so we should check. I recall there's a known issue with the VM being slightly slower than the GraphExecutor due to cache issues, but I suspect that's more likely to be due to D$ effects with the extra indirections around registers or something.

Reply:
Thanks! I always think that simple loop is better than jump approach, it can be checked further. Just now I'm aware about the feature design.